base/commands/serializer/generate.go

```diff
@@ -0,0 +1,179 @@
+//go:build std || serializer
+
+package serializer
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/hazelcast/hazelcast-commandline-client/clc"
+	. "github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/plug"
+)
+
+type GenerateCmd struct{}
+
+const (
+	flagLanguage  = "language"
+	flagOutputDir = "output-dir"
+)
+
+func (g GenerateCmd) Init(cc plug.InitContext) error {
+	cc.SetCommandUsage("generate [schema] [flags]")
+	short := `
+Generates compact serializer from the given schema and for the given programming language. (BETA)
```

Comment:
Could you start the line after the backtick character?
Same comment for `long`.

Reply:
https://github.com/hazelcast/hazelcast-commandline-client/pull/303/commits/049a4a39a4000138b574980e89efadb55b09d852